docker-compose.yml

```diff
@@ -42,13 +42,10 @@ services:
       MYSQL_USER: test
       MYSQL_DATABASE: test
 
-  log-server:
+    log-server:
     depends_on:
       - db
-    image: us.gcr.io/key-transparency/log-server:${TRAVIS_COMMIT}
-    build:
-      context: ../trillian
-      dockerfile: examples/deployment/docker/log_server/Dockerfile
+    image: us.gcr.io/trillian/log-server:latest
```

Comment:
I both love and dislike `:latest`

It both fetches the "latest" image - great. But it also prevents `kubectl apply` from updating the image when "latest" change.  I don't know what the best practices are around this, but "latest" has given me a lot of heartburn keeping the CI environment up-to-date.

Reply:
Yes, I don't recommend `:latest` either. Apologies... It was a shortcut to not having a better way to grab the latest release of Trillian's images.